Core/Core/Extensions/UIApplicationExtension.swift

```diff
@@ -36,8 +45,7 @@ public extension UIApplication {
     }
     
     var windowInsets: UIEdgeInsets {
-        guard let windowScene = UIApplication.shared.connectedScenes.first as? UIWindowScene,
-              let window = windowScene.windows.first else {
+        guard let window = windows?.first else {
```

Comment:
could we use here guard let window = `window` - property added line 18?

Reply:
done